bin/improver-nbhood

```diff
@@ -46,8 +46,12 @@ def main():
                     'to a file with one cube.')
     parser.add_argument('neighbourhood_method', metavar='NEIGHBOURHOOD_METHOD',
                         help='The neighbourhood method to apply in '
-                             'neighbourhood processing. Options: "circular", '
-                             '"square".')
+                             'neighbourhood processing. Options: '
+                             '"probabilities", "percentiles".')
+    parser.add_argument('neighbourhood_shape', metavar='NEIGHBOURHOOD_SHAPE',
```

Comment:
Not all 4 combinations are possible though, so should we do something about that.

Reply:
Fixed in new PR.